File: src/main/java/nl/asdproject2/Writer.java
package nl.asdproject2;

import java.io.*;

public class Writer
{
    public static void write(String output)
    {
        System.out.println(output);
    }

    public static void writeUTF(OutputStream outToServer, String output) throws IOException
    {
        DataOutputStream out = new DataOutputStream(outToServer);
        out.writeUTF(output);
        out.flush();
    }

    public static void writeIn(InputStream inputStream) throws IOException
    {
        DataInputStream in = new DataInputStream(inputStream);
        System.out.println(in.readUTF());
    }
}

File: src/main/java/nl/asdproject2/kotlin/Writer.kt
package nl.asdproject2.kotlin

import java.io.*

object Writer {
    fun write(output: String) {
        println(output)
    }

    @Throws(IOException::class)
    fun writeUTF(outToServer: OutputStream, output: String) {
        val out = DataOutputStream(outToServer)
        out.writeUTF(output)
        out.flush()
    }

    @Throws(IOException::class)
    fun writeIn(inputStream: InputStream) {
        val `in` = DataInputStream(inputStream)
        println(`in`.readUTF())
    }
}
